.circleci/config.yml

```diff
@@ -394,6 +422,49 @@ jobs:
           paths:
             - ./habitat-api
 
+  update_docs:
+    docker:
+      - image: circleci/buildpack-deps:disco
+    steps:
+      - restore_cache:
+          key: docs-{{ .Branch }}
+      - add_ssh_keys:
+          fingerprints:
+            - "18:88:e0:37:b1:b3:7a:23:aa:1e:f7:43:a8:5b:8e:05"
+      - run:
+          name: Update public documentation
+          command: |
+              # Update website
```

Comment:
Add
```
mkdir -p ~/.ssh
touch ~/.ssh/known_hosts
echo "github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==" >> ~/.ssh/known_hosts
```

To deal with the fingerprint issue.

Reply:
what exactly issue?

Reply:
For know add `checkout` step at the beginning. It should take care of that and may be useful for other manipulation with docs.